packages/builders/src/messages/Message.ts

```diff
@@ -31,7 +33,7 @@ export interface MessageBuilderData
 	> {
 	allowed_mentions?: AllowedMentionsBuilder;
 	attachments: AttachmentBuilder[];
-	components: ActionRowBuilder[];
+	components: MessageComponentBuilder[];
```

Comment:
This also includes Buttons and SelectMenus. Those are not top-level components. We need another `MessageTopLevelComponentBuilder` type without those for this.